Documentation/ApiOverview/ContentElements/CustomBackendPreview.rst

```diff
@@ -14,6 +14,65 @@ following sections describe how to achieve that.
 A preview renderer is used to facilitate (record) previews in TYPO3. This
 class is responsible for generating the preview and the wrapping.
 
+The default PreviewRenderer is :php:`StandardContentPreviewRenderer` and handles
+the core's built-in CTypes.
+
+Extend the default PreviewRenderer
+==================================
+
+There's two ways of how you can provide previews for your own custom CTypes:
+via PageTS or event listener.
```

Comment:
```suggestion
via page TSconfig or event listener.
```

Reply:
See https://docs.typo3.org/m/typo3/docs-how-to-document/main/en-us/GeneralConventions/Glossary.html